cluster-scope/overlays/nerc-ocp-test/oauths/cluster_patch.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: config.openshift.io/v1
+kind: OAuth
+metadata:
+  name: cluster
+spec:
+  identityProviders:
+    - name: github
+      mappingMethod: claim
+      type: GitHub
+      github:
+        clientID: 771ea98004d436c6e529
+        clientSecret:
+          name: github-client-secret
+        teams:
+          - ocp-on-nerc/nerc-ops
+          - nerc-logs-metrics
```

Comment:
Based on the existing line, it looks like this should also be fully qualified (`ocp-on-nerc/nerc-logs-metrics`).

Reply:
Oops that's correct, careless copy/paste. Fixed

Reply:
@larsks with this resolved is this PR considered passing review by you?